exercises/concept/shopping/.meta/exemplar.jq

```diff
@@ -0,0 +1,20 @@
+# Task 1
+.name,
+
+# Task 2
+(.ingredients | length),
+
+# Task 3
+(.ingredients[] | select(.item == "sugar") | .amount.quantity),
+
+# Task 4
+(
+  (.ingredients + ."optional ingredients")
+  | map(select(.substitute) | {(.item): .substitute})
```

Comment:
Unless I've missed it, I can't see where I would have learnt about `select` or `map`. I feel both of these need covering in the `introduction.md` (or if they're not actually expected for this exercise, the exemplar shouldn't have them, as the exemplar is the exemplar **based on a student's received knowledge** at this point during the track.